crypto/x509/x509_vfy.c

```diff
@@ -1865,10 +1865,9 @@ int X509_cmp_time(const ASN1_TIME *ctm, time_t *cmp_time)
             return 1;
     }
     i = strcmp(buff1, buff2);
-    if (i == 0)                 /* wait a second then return younger :-) */
-        return -1;
-    else
-        return i;
+    /* 0 is reserved for error, so for equal times, wait a second
```

Comment:
argh: nit.  the "/*" and "*/" on lines by themselves :(

Reply:
I tried to rebel. Didn't work :)

Reply:
That comment is incorrect - we are not waiting a second.
We are defining it to be <= 
The comment in master is more accurate and useful than this misleading comment.